next/components/forms/simple-components/ProgressBar.tsx

```diff
@@ -3,6 +3,7 @@ import { useProgressBar } from 'react-aria'
 
 type ProgressBarBase = {
   type?: 'success' | 'default'
+  label: string
```

Comment:
good intention, but I'm not sure if progress bar should always have a label - i.e. while it's not so in newest design, it used to be used as par of the header without any extra text

in the rest of the (widget) components the label makes more sense